### PR TITLE
Adds rake tasks that add/remove a feature for one/all users/organizations as well as list available features

### PR DESCRIPTION
**Pull request by gfiorav**
This code solves issue [#4677](https://github.com/CartoDB/cartodb/issues/4677)

It adds rake tasks:

rake cartodb:features:disable_feature_for_all_users[feature] # disable feature for all users
rake cartodb:features:disable_feature_for_user[feature,username] # disable feature for a given user
rake cartodb:features:enable_feature_for_all_users[feature] # enable feature for all users
rake cartodb:features:enable_feature_for_user[feature,username] # enable feature for a given user
rake cartodb:features:list_all_features # list all features

CC @javisantana

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5386/))

**Comment by javisantana** [comment]
cc @Kartones

File: lib/tasks/enable_feature.rake [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+      puts "Available features:"
+      FeatureFlag.all.each do |feature|
+        puts "  - #{feature.name}"
+      end
+    end
+
+    private
+    def except(type, value)
+      puts "[ERROR] #{type} '#{value}' does not exist"
```
Comment (Kartones) [inline lib/tasks/enable_feature.rake]: really is worth a one line to just format a string?

I'd instead `raise "some text blablabla"` so it also removes the need of the `break` after the call.

Also we tend to always use parenthesis on method calls, they are optional but in general as they ease readability we favor their use.

Comment (Kartones) [inline lib/tasks/enable_feature.rake]: in fact you can do `raise "xxxx" if user.nil?` and the like to reduce to one-liners your checks

**Comment by Kartones** [comment]
apart from the notes, :+1:

File: lib/tasks/enable_feature.rake (hunk 1 above)
Comment (gfiorav) [inline lib/tasks/enable_feature.rake]: Thanks for the notes!

Please find an updated version at PR.

File: lib/tasks/enable_feature.rake [hunk 2]
```diff
@@ -0,0 +1,125 @@
+namespace :cartodb do
+  namespace :features do
+    desc "enable feature for all users"
+    task :enable_feature_for_all_users, [:feature] => :environment do |t, args|
+      
+      ff = FeatureFlag[:name => args[:feature]]
+      if ff.nil?
```
Comment (Kartones) [inline lib/tasks/enable_feature.rake]: Compression++
```ruby
raise "[ERROR]  Feature '#{args[:feature]}' does not exist"  if ff.nil?
```

Comment (gfiorav) [inline lib/tasks/enable_feature.rake]: Excellent point, indeed!

Find an updated version in the PR.

**Comment by Kartones** [comment]
Now only missing the warning on the rakes... and good to go from my side!

**Comment by gfiorav** [comment]
The warning about central?

**Comment by Kartones** [comment]
yep!

**Comment by gfiorav** [comment]
Comments added.

Will merge as soon as you confirm.

**Comment by Kartones** [comment]
I'd use the term "to use only at development, opensource and custom installs" or something similar, because Central is an internal term, right @javisantana?

Other than that, or if Santana feels is ok this way the message, good to go.

**Comment by gfiorav** [comment]
Will add this comment and commit

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5393/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5396/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5397/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5399/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5400/))

**Comment by javisantana** [comment]
How do I create a feature flag ?

**Comment by gfiorav** [comment]
@javisantana said functionality is now added.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5442/))

**Comment by javisantana** [comment]
thanks!

**Pull request merged**